templates/index.html

```diff
@@ -24,7 +24,7 @@
                             </td>
                             <td>
                                 <a href="//{{ $.Godoc.Host }}/{{ $importPath }}">
-                                    <img src="//{{ $.Godoc.Host }}/{{ $importPath }}?status.svg" alt="GoDoc" />
+                                    <img src="//img.shields.io/badge/godoc-reference-blue?style=for-the-badge" alt="GoDoc" />
```

Comment:
Does going to: https://img.shields.io/badge/godoc-reference-blue%253Fstyle=for-the-badge yield the badge that we want?

Reply:
Yes, it does!